### ISSUE TITLE
Portrait Actor Config Button Missing

### ISSUE THREAD

**Issue by NuptupTDOW** [Saibot393/enhancedcombathud-pf2e#49]
The Actor Configuration option mentioned here: https://wiki.theripper93.com/free/enhancedcombathud#actor-configuration

"Hover over the Top-Right of the Portrait image in the HUD to reveal the configuration button. This button will allow you to configure the HUD for the actor."

Doesn't seem to work/exist.

**Comment by Saibot393** [comment]
Right, this feature doesn't exist for the PF2E version of the hud (yet?).
The reason being that in the D&D 5e version it allows you to set which/how spells are detected (see screenshot), i can not really think of a use case for that for the PF2E version or really any other per character setting that might be usefull.
![image](https://github.com/Saibot393/enhancedcombathud-pf2e/assets/137942782/4c2e6fbc-efe8-401f-b773-c1e2d6286aa5)
